packages/block-editor/src/index.js

```diff
@@ -6,6 +6,7 @@ export {
 	getBorderClassesAndStyles as __experimentalGetBorderClassesAndStyles,
 	useBorderProps as __experimentalUseBorderProps,
 	getColorClassesAndStyles as __experimentalGetColorClassesAndStyles,
+	getTypographyClassesAndStyles as __experimentalGetTypographyClassesAndStyles,
```

Comment:
I know the other util functions were exported as experimental but do you think we could export this as stable? We'll be looking to stabilize all the block support APIs soon.